product_docs/docs/transporter/2_preview/preparing_db/preparing_oracle_source_databases.mdx

```diff
@@ -0,0 +1,279 @@
+---
+title: "Preparing Oracle source databases"
```

Comment:
Please do check over all of these...the ones that follow the letter V (logs, I believe) I just wanted to double check on.

Reply:
I'm going to revert this changes. V$... & V_$.... are oracle tables name.